lib/presentation/localizations/app_localizations.dart

```diff
@@ -2160,6 +2160,12 @@ class AppLocalizations {
     );
   }
 
+  String get search_in_upload_requests {
```

Comment:
Should be "Search in Upload Request Entry". Because of we will have a "Search in Upload Request" for recipient (Upload Request) in the other tickets. Please also add string to all languages too